compress_jointcal_cfht_test_data.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
```

Comment:
Do we need to propagate `PYTHONPATH` in order to get `astropy`?  Or is it installed as part of Anaconda and therefore python will set the paths appropriately?